### PR TITLE
Onboard development

### PR DESCRIPTION
**Pull request by AlfaHotel**

### CODE REVIEW

File: master_module_ble/main.c [hunk 1]
```diff
@@ -419,7 +345,8 @@ static uint32_t adv_report_parse(uint8_t type, data_t * p_advdata, data_t * p_ty
 static void on_ble_evt(ble_evt_t * p_ble_evt)
 {
     uint32_t         err_code;
-    static uint16_t  service_uuid_list[3] = {SHORT_SERVICE_RELAYR_UUID, BLE_UUID_DEVICE_INFORMATION_SERVICE, BLE_UUID_BATTERY_SERVICE};
+    // static uint16_t  service_uuid_list = {SHORT_SERVICE_RELAYR_UUID, BLE_UUID_DEVICE_INFORMATION_SERVICE, BLE_UUID_BATTERY_SERVICE};
```
Comment (marogi) [inline master_module_ble/main.c]: Dead code can be removed

File: master_module_ble/onboard.h [hunk 2]
```diff
@@ -13,47 +13,12 @@
...
-    ONBOARD_CHAR_INDEX_SOUND_BRIDGE_IR_PASS  = 1,
-    ONBOARD_CHAR_INDEX_WIFI_SSID             = 2,
-    ONBOARD_CHAR_INDEX_WIFI_PASS             = 3,
-    ONBOARD_CHAR_INDEX_MASTER_MODULE_ID      = 4,
-    ONBOARD_CHAR_INDEX_MASTER_MODULE_SEC     = 5,
-	  ONBOARD_CHAR_INDEX_MASTER_MODULE_URL     = 6
-}
-onboard_characteristics_t;
+/**@brief  Default values for sensors passkeys. These values are used if corresponding block of persistent storage is empty. */
+// static const uint8_t DEFAULT_SENSOR_PASSKEY[PASSKEY_SIZE+2]   = {0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x00, 0x00};
```
Comment (marogi) [inline master_module_ble/onboard.h]: Which key should be the default?

File: master_module_ble/main.c (hunk 1 above)
Comment (AlfaHotel) [reply master_module_ble/main.c]: done

File: master_module_ble/onboard.h (hunk 2 above)
Comment (AlfaHotel) [reply master_module_ble/onboard.h]: done

**Pull request merged**